api/graphql/schema.graphql

```diff
@@ -37,17 +37,25 @@ type User {
     poolCandidates: [PoolCandidate] @hasMany # Pools candidates that a user owns
 }
 
+type PoolOwnerPublicProfile {
```

Comment:
Since this is essentially a public form of the user type I wonder if it would make more sense to call this **UserPublicProfile**.  We could reuse it outside of the context of owning a pool, like a user search tool.  So like `owner: User` there would be `ownerPublicProfile: UserPublicProfile`

Reply:
Overall, I do like this approach, though.  I think this pattern could be reused in other places.

Reply:
Was having a sidebar discussion with @mnigh and he found this thread [https://github.com/nuwave/lighthouse/issues/325#issuecomment-421827472](https://github.com/nuwave/lighthouse/issues/325#issuecomment-421827472).  It might be worth looking into not duplicating the model types and instead using directives to protect individual fields at that more granular level.  It might result in a smaller schema that is more DRY.